kofta/src/vscode-webview/components/UserProfile.tsx

```diff
@@ -115,7 +117,23 @@ export const UserProfile: React.FC<UserProfileProps> = ({
           following
         </button>
       </div>
-      <div className={`mb-4`}>{profile.bio}</div>
+      <div className="mb-4">
+        {profile.bio?.split(" ").map((chunk, i) => {
+          return linkRegex.test(chunk) ? (
+            <a
+              key={i}
+              href={normalizeUrl(chunk)}
+              target="_blank"
+              rel="noreferrer"
+              className="text-tmpC4 p-0"
```

Comment:
I think `tmpC4` is gone

Reply:
Yeah might want to double-check the css

Reply:
when i wrote it the new breaking or wasn't merged. so didn't know, fixing.